app/api/relationships.v2/contracts/RelationshipsDataSource.ts

```diff
@@ -8,6 +8,7 @@ export interface RelationshipsDataSource {
   delete(_ids: string[]): Promise<void>;
   deleteAll(): Promise<void>;
   exists(_ids: string[]): Promise<boolean>;
+  getAll(): ResultSet<Relationship>;
```

Comment:
I saw this is only used in a test. I think that until `getAll` is an application requirement it shouldn't be in the contract. Since that test is an integration test, and you are already instantiating a Mongo implementation, it's safe to say that you can simply use the method from the implementation (which in fact you are) and not include it here :)